pkg/kritis/review/validating_transport.go

```diff
@@ -141,11 +142,11 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 			continue
 		}
 		keyId := rawAtt.Signature.PublicKeyId
-		if err = host.VerifySignature(keys[keyId], string(decodedSig)); err != nil {
+		if err = avt.ContainerHost.VerifySignature(keys[keyId], string(decodedSig)); err != nil {
```

Comment:
OK, I think now I understand what you're trying to do.  I think you're trying to create an injection point here, so you can mock signature verification in unit tests.  Right?

I think that's a good idea, but requires a little different approach.  I think you'll end up having to define the crypto interface that we were discussing, and then mocking it up for tests.  I don't think ContainerHost is the thing you want to mock here.